src/Symfony/Component/HttpKernel/composer.json

```diff
@@ -63,6 +63,7 @@
         "symfony/http-client-contracts": "<2.5",
         "symfony/mailer": "<5.4",
         "symfony/messenger": "<5.4",
+        "symfony/service-contracts": "<3.2",
```

Comment:
@kbond why was this added? is there another way to remove this conflict?

this makes it not possible for us to upgrade to symfony 6.4 because this forces a requirement for `psr/container ^2.0`, and we are still on v1 because the latest version of `laminas/laminas-servicemanager` requires v1:
https://github.com/laminas/laminas-servicemanager/blob/3.23.x/composer.json#L26

this is sort of the same issue as https://github.com/api-platform/core/pull/5811

Reply:
/cc @nicolas-grekas

Reply:
Sorry @kbond, tagged you as it looked like your commit.

Reply:
This is needed because the test case uses the $type argument of SubscribedService, which exists since 3.2.
PR welcome to change the test case and relax this.

Reply:
thanks